src/sass/Fabric.Icons.Output.scss

```diff
@@ -34,6 +34,10 @@
   @include ms-Icon--l;
 }
 
+// @todo: 'DynamicsCRMLogo' class is deprecated, will be removed in a future release
+// @todo: 'CarotRightSolid8' class is deprecated, will be removed in a future release
```

Comment:
I see you've moved the comments up here, but where are the deprecated classes themselves?